src/data/BlueHillAveCorridor.ts

```diff
@@ -1,4 +1,4 @@
-export const BlueHillAveCorridor: MapGeoJsonData = {
+export const BlueHillAveCorridor: GeoJson = {
```

Comment:
The geojson from, say `/boundaries` folder, needs to live in `/src/data` folder. Because these geojson-like js files are actually js files (they all start with var {some variable name}) there wasn't an easy way to import the geojson-like data from outside of src. I attempted to have the `/boundary` in `/public` and see if CRA's 'escape hatch' would work https://create-react-app.dev/docs/using-the-public-folder/ it really only works well with simple asset types and not so much unexported js variables.

I still think that these js files should either:
a. be converted from .js file to .geojson or .json (and thus removing the variable declaration in each of them) and keeping them in /public/boundary or
b. move them into /src/data (as CRA intends with data files) and convert to .ts file and give it a type

This PR is currently doing the latter option so to stay coupled to the type system

Reply:
https://github.com/codeforboston/mattapan-mapping/issues/75

Reply:
Yep yep! As we spoke about, lets document the final decisions in #75 maybe